.github/workflows/docker-publish.yml

```
@@ -1,32 +1,75 @@
-        run: docker buildx build --push --platform linux/amd64,linux/arm64 --build-arg NETWORK=regtest -t boltz/web-app:latest .
+    build-amd64:
+        name: Build AMD64
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check out code
+              uses: actions/checkout@v6
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v3
+              with:
+                  install: true
```

actionlint (1.7.11)

[error] avoid using deprecated input "install" in action "docker/setup-buildx-action@v3": "docker buildx install" command is deprecated and will be removed in a future release, use BUILDX_BUILDER environment variable instead

(action)